### PR TITLE
oa: add shared community token

### PR DESCRIPTION
**Pull request by missinglink**
This PR adds a default OpenAddresses token which can be used to authenticate downloads to `batch.openaddresses.io`.

One of the changes from the legacy `results.openaddresses.io` domain to the new `batch` domain was the introduction of token credentials for downloads. The purpose of this change was to better track and prevent abuse of the download server.

Requiring each new Pelias user to register their own account at https://batch.openaddresses.io would add some unnecessary friction to the new user onboarding experience with Pelias.

After discussing this with the OA team we settled on this idea of having a *default shared credential* which will 'just work' out-of-the-box.

Using this default token comes with some caveats that users should be aware of:
- throttling/bandwidth limits will be shared between all users of the shared token
- abuse of the token may result in temporary suspension or termination of the credential.
- as such, use of the shared token comes with no guarantees

Anyone serious about their installation (ie. beyond just 'playing around with it') **should register their own account and mint their own individual token** at https://batch.openaddresses.io. Doing so will mitigate the issues above by providing individual tracking and abuse prevention measures by the OA team.

Additionally, there are some features which are only available to financial sponsors of the OA project, for example, as a `>$100/mo` sponsor, Geocode Earth will be able to access downloads directly from AWS S3. We will aim to automatically detect and support these 'sponsor-level features'. Please consider financially supporting the OA project if your business depends on their work (the bandwidth bills alone are significant).

Once you've minted your own token, it's a simple matter of editing your `pelias.json` file and setting `imports.openaddresses.token={YOUR_TOKEN}`, Pelias will pick up your individual token and use it instead of the default token.

### CODE REVIEW

**Review by Joxit — APPROVED** [review]

**Comment by Joxit** [comment]
We should link this PR somewhere in [pelias/openaddresses](https://github.com/pelias/openaddresses) (print during download when the token is the default one ?)
This is very interesting and should be as visible as possible.

**Comment by orangejulius** [comment]
Would it make more sense to put a default community token in the OA repository code itself, and then have the code look to `pelias.json` for an alternate key to override the default if present?

Otherwise, if and when we have to rotate the shared OA key due to inevitable abuse, we have to bump the version of the pelias/config package and updated it in the OA repository. No other part of Pelias will likely care about the OA token, so that seems like a bit of extra package churn for no reason.

If we keep the token in the OA repository, then it's just a single PR and everything is done.

**Comment by missinglink** [comment]
@orangejulius I considered doing it like that, but in the end felt like it was preferable to have the default set in the same place as a user would need to edit to override it.

I often refer to the 'config defaults' in this repo for reference for naming conventions, so it serves as a form of documentation.

I'm not really fussed about having two PRs, this one is also nice for documentation purposes.

The package churn is a potential issue if we end up doing regular key cycling at a later date (I'm hoping we never need to cycle the key at all) but I don't think that's an immediate problem.

tl;dr I think both methods are valid and work but find this method more 'idiomatic' in that all config variables are in the same place, while also serving as a form of documentation.

Thoughts?

**Review by orangejulius — APPROVED** [review]
Oops, I let this one slip, but yeah, keeping the token here as documentation makes sense. Hopefully we don't have to change this too often.
